docs/tutorials/overview.md

```diff
@@ -3,3 +3,5 @@
 In this section you can find step-by-step guides that help you accomplish specific tasks.
 
 👈 Please use the side-navigation on the left in order to browse the different topics.
+
+If there are any specific tutorials that you would like to see please let us know on the [Aqua Open Source Slack](https://slack.aquasec.com/).
```

Comment:
I suggest not to rely on slack too much, why not a documentation labeled issue?

Reply:
oki